### PR TITLE
fix: load sentry on all servers

### PR DESCRIPTION
**Pull request by williamrusdyputra**

### CODE REVIEW

File: servers/index-minertoken.js [hunk 1]
```diff
@@ -4,23 +4,38 @@
...
+  Sentry.init({
+    dsn: process.env.SENTRY_DSN,
+    integrations: [
+      nodeProfilingIntegration()
+    ],
+    tracesSampleRate: 0.2,
+    profilesSampleRate: 0.2
+  })
+}
+
```
Comment (georgeciubotaru) [inline servers/index-minertoken.js:25]: not DRY

**Review by georgeciubotaru — CHANGES_REQUESTED** [review]
See this one: https://docs.sentry.io/platforms/python/integrations/fastapi/

**Comment by williamrusdyputra** [comment]
> See this one: https://docs.sentry.io/platforms/python/integrations/fastapi/

I think you mean for transformer PR?

**Comment by georgeciubotaru** [comment]
@williamrusdyputra yes

**Review by georgeciubotaru — APPROVED** [review]

**Pull request merged**